MeshLib/Elements/Element.h

```diff
@@ -89,7 +89,7 @@ class Element
 	virtual const Element* getFace(unsigned i) const = 0;
 
 	/// Returns the ID of the element.
-	virtual std::size_t getID() const { return this->_id; }
```

Comment:
Not sure on removal of the virtual function specifier here.
I think it is better to add a final specifier here such that `getID()` cannot be overridden by any derived class. Without any specifiers one can implement `getID()` in a derived class with funny consequences.